agent/engine/docker_image_manager_test.go

```diff
@@ -776,8 +776,8 @@ func TestImageCleanupCannotRemoveImage(t *testing.T) {
 	imageState.PulledAt = time.Now().AddDate(0, -2, 0)
 	imageState.LastUsedAt = time.Now().AddDate(0, -2, 0)
 
-	client.EXPECT().RemoveImage(container.Image, removeImageTimeout).Return(errors.New("error removing image")).AnyTimes()
-	imageManager.removeUnusedImages()
+	client.EXPECT().RemoveImage(gomock.Any(), container.Image, removeImageTimeout).Return(errors.New("error removing image")).AnyTimes()
+	imageManager.removeUnusedImages(context.TODO())
```

Comment:
Same comment as in the other file about using cancellable context everywhere in this file